app/src/pages/inside/dashboardPage/widgets/components/testsTableWidget/pTypes.js

```diff
@@ -0,0 +1,52 @@
+import { string, node, shape, arrayOf, bool, number, oneOf } from 'prop-types';
+
+export const PTLaunch = shape({
+  id: string.isRequired,
+  name: string.isRequired,
+  number: string.isRequired,
+  issueType: string,
+});
+
+export const PTStatus = oneOf(['FAILED', 'PASSED', 'SKIPPED']);
```

Comment:
Use constants from common/constants/launchStatuses.js and str.toUppercase()

Reply:
Fixed